conductr_cli/exceptions.py

```diff
@@ -37,6 +37,14 @@ def __str__(self):
         return repr(self.value)
 
 
+class ResolverError(Exception):
```

Comment:
Is the `ResolverError` used? I can't seem to find where `ResolverError` is raised from.

I guess if the `ResolverError` is meant to be raised by one of the resolver, the `load()` method in the `conduct_load.py` need to have validation handler for this (i.e. similar to `@validation.handle_bintray_resolution_error`)